libmscore/instrument.cpp

```diff
@@ -772,7 +789,6 @@ void Channel::read(XmlReader& e, Part* part)
             }
       if (128 == _bank && "zerberus" == _synti.toLower())
             _bank = 0;
-      updateInitList();
```

Comment:
Shouldn't we set `_mustUpdateInit` flag there? Read operation would probably change some related data, and the code above operates on `Channel` members directly so this flag won't get updated automatically.

Reply:
Yes, good spot. Thanks.